packaging/ubuntu-16.04/control

```diff
@@ -111,6 +111,14 @@ Depends: snapd (= ${binary:Version}), ${misc:Depends}
 Description: Transitional package for snapd
  This is a transitional dummy package. It can safely be removed.
 
+Package: snap-confine-debug
```

Comment:
Why do we install it into an extra pacakge instead of just shipping it as part of snapd?

Reply:
Because nobody outside of the team would need to use it and it is somewhat larger than the regular build.